tests/comparisons/Plugin/testMainUpdateComposer.json

```diff
@@ -1,12 +1,12 @@
 {
     "autoload": {
         "psr-4": {
-            "ComposerExample\\": "./plugins/ComposerExample/src"
+            "ComposerExample\\": "/tmp/tests/BakedPlugins/ComposerExample/src"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "ComposerExample\\Test\\": "./plugins/ComposerExample/tests"
+            "ComposerExample\\Test\\": "/tmp/tests/BakedPlugins/ComposerExample/tests"
```

Comment:
The path used above is related to the fact that the plugin path in the Test Instance is set to /tmp/tests/BakedPlugins, on a productive environment it would look like:
'"Test6\\Test\\": "./src/Plugin/Test6/tests"